docker/opensource.md

```diff
@@ -87,5 +87,102 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
 # phpunit
 ```
 
+## Cosmo-Tech
+{id: docker-python-Cosmo-Tech}
+
+
+* [Cosmo-Tech](https://github.com/Cosmo-Tech/CosmoTech-Acceleration-Library)
+
+```
+$ git clone https://github.com/Cosmo-Tech/CosmoTech-Acceleration-Library.git
+$ cd CosmoTech-Acceleration-Library
+$ docker run -t -i --name cosmotech-acceleration-library-dev -v /opt/CosmoTech-Acceleration-Library -v /C:/Users/YourUserName/CosmoTech-Acceleration-Library:/opt python:3.11 bash
+
+# cd opt
+# pip install -r requirements.txt
+# pip install pytest
+# pytest
+```
+
+```
+$ docker container start -i cosmotech-acceleration-library-dev
+```
+
+## mobility
+{id: docker-python-mobility}
+
+
+* [mobility](https://github.com/mobility-team/mobility)
+```
+$ git clone https://github.com/mobility-team/mobility.git
+$ cd mobility
+$ docker run -t -i --name mobility-dev -v /opt/mobility -v /C:/Users/YourUserName/mobility:/opt python:3.9 bash
```

Comment:
as above

Reply:
the demand was for python_requires=">=3.9"
So I chose 3.9, becasue somtimes the update version did errors in the tests